### PR TITLE
Reusable templates

### PR DESCRIPTION
**Pull request by e-davidson**

### CODE REVIEW

File: projects/table-builder/src/lib/components/column-builder/column-builder.component.ts [hunk 1]
```diff
@@ -17,41 +18,33 @@ import { map, startWith, tap } from 'rxjs/operators';
...
 
-  template: TemplateRef<any>;
-  transform: (o: any, ...args: any[])=> any ;
 
-  constructor( private transformCreator: TransformCreator, private table: MatTable<any>, 
+  constructor(
+    private transformCreator: TransformCreator,
+    private table: MatTable<any>,
     public state: TableStore,
+    private templateService: TableService,
```
Comment (NachmanBerkowitz) [inline projects/table-builder/src/lib/components/column-builder/column-builder.component.ts:37]: templateService or TableService

File: projects/table-builder/src/lib/pipes/transform-pipe.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
```
Comment (NachmanBerkowitz) [inline projects/table-builder/src/lib/pipes/transform-pipe.ts:1]: is this being used

**Review by NachmanBerkowitz — CHANGES_REQUESTED** [review]

File: projects/table-builder/src/lib/pipes/transform-pipe.ts (hunk 2 above)
Comment (e-davidson) [reply projects/table-builder/src/lib/pipes/transform-pipe.ts]: no, removed.

File: projects/table-builder/src/lib/components/column-builder/column-builder.component.ts (hunk 1 above)
Comment (e-davidson) [reply projects/table-builder/src/lib/components/column-builder/column-builder.component.ts]: renamed

**Review by NachmanBerkowitz — APPROVED** [review]

**Pull request merged**